pyas2/models.py

```diff
@@ -39,6 +43,36 @@ class PublicCertificate(models.Model):
         verbose_name=_('Verify Certificate'), default=True,
         help_text=_('Uncheck this option to disable certificate verification.'))
 
+    certificate_valid_from = models.DateTimeField(null=True, blank=True)
+    certificate_valid_to = models.DateTimeField(null=True, blank=True)
+
+    def extract_info(self):
```

Comment:
Also it would be good to save the certificate serial number.

Reply:
Yes, can be done too. Not sure what field type to use for this though. BinaryField, FloatField ? Seems the serial would be a max 20 byte value.

Reply:
use CharField

Reply:
OK to do: 
serial_number = models.CharField(max_length=64, null=True, blank=True) 
?

Reply:
lets call it serial_number